.changeset/strange-starfishes-joke.md

```diff
@@ -0,0 +1,5 @@
+---
+"@nomicfoundation/hardhat-ledger": patch
+---
+
+Improved the `hardhat-ledger` README [7585](https://github.com/NomicFoundation/hardhat/pull/7585)
```

Comment:
I am going to drop this changelog, I think we put together a PR when we are ready to release and put one changelog in there.